libs/ui/src/precinct_scanner_tally_report.test.tsx

```diff
@@ -56,9 +56,17 @@ test('renders as expected for all precincts in a general election', () => {
   );
   expect(screen.queryByText('Party')).toBeNull();
   screen.getByText('Official Polls Closed Report for All Precincts');
-  screen.getByText('General Election');
-  screen.getByText(
-    /Polls closed on Sep 19, 2021, 11:05 AM. Report printed on Sep 19, 2021, 11:06 AM./
+  const electionTitle = screen.getByText('General Election:');
+  expect(electionTitle.parentElement).toHaveTextContent(
+    'General Election: Tuesday, November 3, 2020, Franklin County, State of Hamilton'
+  );
+  const eventDate = screen.getByText('Polls Closed:');
+  expect(eventDate.parentNode).toHaveTextContent(
+    'Polls Closed: Sep 19, 2021, 11:05 AM'
+  );
+  const printedAt = screen.getByText('Report Printed:');
+  expect(printedAt.parentElement).toHaveTextContent(
+    'Report Printed: Sep 19, 2021, 11:06 AM'
```

Comment:
I realize I forgot to add test assertions for the `Scanner ID:` section, which should be added for consistency.

Reply:
Done.